common/src/Microsoft.Azure.IIoT.Core/src/Messaging/EventHub/Services/EventHubDeviceEventHandler.cs

```diff
@@ -52,7 +56,7 @@ public sealed class EventHubDeviceEventHandler : IEventHandler {
         /// <inheritdoc/>
         public async Task OnBatchCompleteAsync() {
             foreach (var handler in _used) {
-                await Try.Async(_handlers[handler].OnBatchCompleteAsync);
+                await Try.Async(_handlers[handler]!.OnBatchCompleteAsync);
```

Comment:
Is this intentended?

Reply:
It was an attempt to get rid of the NullReferenceException we had, I will revert it.